lerobot/common/policies/tdmpc/modeling_tdmpc.py

```diff
@@ -112,13 +112,12 @@ def __init__(
             config.output_shapes, config.output_normalization_modes, dataset_stats
         )
 
-    def save(self, fp):
-        """Save state dict of TOLD model to filepath."""
-        torch.save(self.state_dict(), fp)
+        image_keys = [k for k in config.input_shapes if k.startswith("observation.image")]
+        # Note: This check is covered in the post-init of the config but have a sanity check just in case.
+        assert len(image_keys) == 1
+        self.input_image_key = image_keys[0]
```

Comment:
```suggestion
        image_keys = [k for k in config.input_shapes if k.startswith("observation.image")]
        # Note: This check is covered in the post-init of the config but have a sanity check just in case.
        # TODO(alexander-soare): make diffusion compatible with multiple image keys
        assert len(image_keys) == 1
        self.input_image_key = image_keys[0]
```

Reply:
or raise NotImplementedError

Reply:
Done.